callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsBridgeKeepAliveManager.java

```diff
@@ -123,6 +123,13 @@ public void shutDownKeepAliveSender() {
 	public void startKeepAliveSender(String authToken) {
 		logger.info("Starting keepAlive Sender");
 		this.token = authToken;
+		
+		//make sure there is no thread running already
+		scheduler.shutdownNow();
```

Comment:
if we shut it down here, then I guess the next line call:
```scheduler.scheduleAtFixedRate()``` might throw exception.